shared/desktop/renderer/index.js

```diff
@@ -93,6 +93,13 @@ function setupApp(store) {
     })
   })
 
+  // After a delay, see if we're connected, and try starting keybase if not
+  setTimeout(() => {
```

Comment:
The callback persists, so 'kb-service-check' will happen on a future disconnect too. I think the main case we wanted to cover is if the services weren't ever started at all, so maybe we don't need to repeat it? Because if it's a crash, the watchdog should restart it already.